site/docs/try/configuration.md

```diff
@@ -19,10 +19,11 @@ docker run \
 
 ### Core Settings
 
-| Property                                  | Default values | Type      | Description                                                               |
-|-------------------------------------------|----------------|-----------|---------------------------------------------------------------------------|
-| `nessie.server.default-branch`            | `main`         | `String`  | Sets the default branch to use if not provided by the user.               |
-| `nessie.server.send-stacktrace-to-client` | `false`        | `boolean` | Sets if server stack trace should be sent to the client in case of error. |
+| Property                                  | Default values | Type      | Description                                                                                                                                                                                                                                                                                                                                       |
+|-------------------------------------------|----------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `nessie.server.default-branch`            | `main`         | `String`  | Sets the default branch to use if not provided by the user.                                                                                                                                                                                                                                                                                       |
+| `nessie.server.send-stacktrace-to-client` | `false`        | `boolean` | Sets if server stack trace should be sent to the client in case of error.                                                                                                                                                                                                                                                                         |
+| `nessie.server.access-checks-batch-size`  | `100`          | `int`     | The number of entity-checks that are grouped into a call to `BatchAccessChecker`. The default value is quite conservative, it is the responsibility of the operator to adjust this value according to the capabilities of the actual authz implementation. Note that the number of checks can be exceeded depending on the context of the checks. |
```

Comment:
just to confirm: the last sentence means there might be more elements in a batch than configured?

Reply:
Yes

Reply:
A few, not "tons"